ArduPlane/Parameters.cpp

```diff
@@ -585,8 +585,8 @@ const AP_Param::Info Plane::var_info[] = {
     GSCALAR(initial_mode,        "INITIAL_MODE",     MANUAL),
 
     // @Param: LIM_ROLL_CD
-    // @DisplayName: Maximum Bank Angle
-    // @Description: The maximum commanded bank angle in either direction
+    // @DisplayName: Maximum bank angle commanded in modes with stabilized limits. Increase this value for sharper turns, but decrease to prevent accelerated stalls.
+    // @Description: 
```

Comment:
Description is blank, display name too long.

Reply:
Fixed these two issues. Thanks for reviewing!

Reply:
I have more to read.. Lots of changes. Most/All are good. thanks for this!